packages/website/src/components/Announcement.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import Text from './Text';
+import { H4 } from './Heading';
+import Box from './Box';
+import Section from './Section';
+
+interface AnnouncementProps {
+  location: Object;
+}
+
+const AnnouncementBox = styled(Box)`
+  background: #ffe9bf;
+  width: 60%;
+  padding: ${({ theme }) => theme.space.medium} ${({ theme }) => theme.space.large};
```

Comment:
Can you please change this padding to the following? (It's to adjust for the paragraph margin)

```
padding: ${({ theme }) => theme.space.large} ${({ theme }) => theme.space.large} ${({ theme }) => theme.space.xSmall} ${({ theme }) => theme.space.large};
1.6103rem 1.6103rem 0.3818rem 1.6103rem
```

Reply:
Will do